playbook/app/pb_kits/playbook/pb_stat_value/_stat_value.jsx

```diff
@@ -25,6 +25,7 @@ const StatValue = (props: StatValueProps) => {
       return (
         <Title
             size={1}
+            tag="span"
```

Comment:
Was this shipping as a "div" by default previously? It'll be interesting to see how the testing of this looks, and see if any default user agent styles are brought in that **might break things with "span". Anything you noticed?

Reply:
I have noticed nothing so far. This is a good point and case for using our `reset.css` in Nitro however.